chart-installation/generate_map_files/mapgen/layer.py

```diff
@@ -153,12 +154,26 @@ def __init__(self, layer_parent, geom_type, classes):
         self.geom_type = geom_type
         self.classes = classes
 
+    def get_human_readable_geomtype(self):
+        t = str(self.geom_type)
```

Comment:
You should just compare to the enum values; that will let the interpreter catch typos

Reply:
the proposed change is now applied in branch